vendor/bat-native-ads/src/bat/ads/internal/ml/pipeline/text_processing/text_processing_unittest.cc

```diff
@@ -109,7 +109,10 @@ TEST_F(BatAdsTextProcessingPipelineTest, TestLoadFromJson) {
   // Act
   ASSERT_TRUE(json_optional.has_value());
   const std::string json = json_optional.value();
-  bool load_success = text_processing_pipeline.FromJson(json);
+  absl::optional<base::Value> value_optional = base::JSONReader::Read(json);
```

Comment:
nit: Use ParseJson, it makes the code more clear.
https://source.chromium.org/chromium/chromium/src/+/main:base/test/values_test_util.h;l=90;bpv=1;bpt=1?q=values_test

Reply:
fixed